jbang-catalog.json

```diff
@@ -9,6 +9,10 @@
       "script-ref": "properties.java",
       "description": "Dump table of System properties"
     },
+    "deps": {
+      "script-ref": "deps.java",
+      "description": "Show the latest available dependency updates"
```

Comment:
I would improve the description slightly, because I think for regular users it's not clear what this is for.
Perhaps "Show the latest available dependency updates for a given script"? (I know @maxandersen doesn't really like to refer to things as "scripts" but in the context of JBang it's clearer than something generic like "code" IMO)

Reply:
I see now that the script itself also has a description, "Analyze JBang script dependencies (//DEPS)", it might be better to just make them the same text? (You choose which one, both are fine with me)

Reply:
Changed the description slightly and made it the same in both places.